### PR TITLE
yeoman-generator 5.0

### PR DESCRIPTION
**Pull request by neurolag**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://yeoman.io/generators/>, <https://github.com/yeoman/generator/releases/tag/v5.0.0>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by manuth** [comment]
Guys, please...?

**Review by fernandopasik — APPROVED** [review]

File: types/yeoman-generator/index.d.ts [hunk 1]
```diff
@@ -872,6 +892,29 @@ declare class Generator<T extends Generator.GeneratorOptions = Generator.Generat
...
+    /**
+     * Adds dependencies to the destination `package.json`.
+     *
+     * @param dependencies
+     * The packages to add.
+     *
+     * @returns
+     * The newly added dependencies.
+     */
+    addDependencies(dependencies: { [pkg: string]: string } | string | string[]): Promise<{ [pkg: string]: string }>;
```
Comment (fernandopasik) [inline types/yeoman-generator/index.d.ts:905]: just one tiny suggestion, why not use `Record<string,string>` instead of `{ [pkg: string]: string }`?

Comment (manuth) [reply types/yeoman-generator/index.d.ts]: Alright!
I replaced all objects with `Record`s now

Thanks für the advice 😄

**Review by fernandopasik — APPROVED** [review]

**Comment by manuth** [comment]
Ready to merge

**Pull request merged**